src/button/__tests__/button.test.tsx

```diff
@@ -91,6 +92,112 @@ describe('Button Component', () => {
     });
   });
 
+  describe('disabled with reason', () => {
+    beforeEach(() => {
+      jest.spyOn(ReactDOM, 'createPortal').mockImplementation((element: any) => element);
```

Comment:
Tooltip component uses react portal. This mock ensures that tooltips is rendered within root element to make using test-utils possible